.eslintrc.cjs

```diff
@@ -10,6 +10,7 @@ module.exports = {
 		"plugin:regexp/recommended",
 		"prettier",
 	],
+	ignorePatterns: ["/docs", "/lib"],
```

Comment:
Why not add this to `.eslintignore`?

Reply:
ah right, I forgot about that file.
I tend to just use this in my projects.